addons/pos_adyen/static/src/app/utils/payment/payment_adyen.js

```diff
@@ -202,12 +220,69 @@ export class PaymentAdyen extends PaymentInterface {
         }
     }
 
+    _adyenHandlePaymentStatus(response, paymentLine, resolve, pollingIntervalId) {
+        const transactionStatus = response.SaleToPOIResponse.TransactionStatusResponse.Response;
```

Comment:
undefined

undefined

Occured on 2025-05-23 09:44:36 GMT

TypeError: Cannot read properties of undefined (reading 'TransactionStatusResponse')
    at Proxy._adyenHandlePaymentStatus (https://79748781-master-all.runbot204.odoo.com/web/assets/384db91/point_of_sale.assets_prod.min.js:17292:126)
    at https://79748781-master-all.runbot204.odoo.com/web/assets/384db91/point_of_sale.assets_prod.min.js:17294:59